README.md

```diff
@@ -1,5 +1,35 @@
 # egghead-articles
 
+models at http://www.objgen.com/json/models/Vzz
+
+model used at https://www.json-generator.com/ for the current data
+```
+[
+  '{{repeat(13)}}',
+  {
+ articleId: '{{guid()}}',
+  author: '{{firstName()}} {{surname()}}',
+  category: '{{random("typescript", "javascript", "react")}}',
+  tags: [
+      '{{repeat(3)}}',
+      '{{random("typescript", "javascript", "react")}}'
+    ],
+  isDraft: false,
```

Comment:
with our api everything is "snake" case, so this would be `is_draft` for example.

Reply:
I'll fix them in the next push